### DISCUSSION TITLE
How many people are using this?

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#23] [General]
Jlai,

Your dashboard is very good, I am curious do you know how many people are using it?

**Comment by jlai** [comment]
The site doesn't track individual users, so I don't know. Not that many. I'd guess a dozen.

**Reply by gramapp** [reply]
Moderators on the Fitbit community keep deleting every entry I make referencing your dashboard. I already wrote in another thread that I can't find your dashboard when googling it, only with other search engines. So I can't spread the word :-( Maybe I need to mention it on Reddit; there are several threads discussing dashboard discontinuity, but I don't have an account there.

I use the dashboard daily. Keep up the good work. the word of mouth will spread it.

**Reply by jlai** [reply]
I tried posting on reddit and it got held for moderation, probably because I was a new poster and didn't have enough karma. And the moderator doesn't seem to be active. I might try again sometime soon.

I'm probably going to refrain from posting on Fitbit community if they take down my post again. If they don't want it there, not much we can do besides find another forum.

**Reply by JimmyG1814** [reply]
> The site doesn't track individual users, so I don't know. Not that many. I'd guess a dozen.

I absolutely love what you have done here to continue to use our Fitbit devices with online access!! I do hope that many more than your estimate of a dozen or so is understated.  I too have posted a link to your hard work on the community comments.  So far they haven't deleted my post.  The link I provide is directly to the Dashboard not to gitbub.com though.  I use the Dashboard multiple times daily for tracking my food and calorie intake.  I do hope that you will continue to provide this to us, even though Google/Fitbit is trying is apparently trying desperately to make it more difficult for you.  I will also try and spread the word via other social media groups such as Reddit!

You are a Godsend my Fitbit friend 💯

**Reply by jlai** [reply]
I added something to count the number of users without tracking individual users. Currently about 200 people have used it this week.

**Reply by JimmyG1814** [reply]
I will endorse all of the previous comments regarding just how thankful I am that you have provided this and **very good I might add,** solution to we disgruntled Fitbit users!!!! Weren't it for Fitbit trackers, online and the app I would never would have lost the amount of weight that I did, by closely monitoring my exercise and eating.  Thanks again to you and all those who have been a part of the hours/days/weeks you have put in to provide this!!!!

**Reply by TLeafTO** [reply]
I've posted a few times on Fitbit community about "Fitness Dashboard" as a great replacement for the discontinued Google Web Dashboard. A few people have thanked me for the link https://dashboard.exercise.quest/ but many people just keep complaining without reading and following the solution that there is.
Thank you  very much for creating "Fitness Dashboard"!

**Reply by gramapp** [reply]
> I added something to count the number of users without tracking individual users. Currently about 200 people have used it this week.

Wow positively impacting at least 200 ppl. I wish I could be this helpful. Keep up the good work!!!!

There are more posts on the fitbit forum about this, it must be getting traction.

**Comment by steviedean** [comment]
I'm loving what you've done here also...am going to be building a step based fitness site (blog style), will make sure I push this wonderful solution...please keep doing your thing, it's awesome!!!

**Comment by mscatz-9** [comment]
Well, count me among those who are using the dashboard.  I absolutely love it!!  So much better than what we had from Google/Fitbit.

**Comment by picander78** [comment]
Hi count me too, i love this site! I just signed in on github to thank you!

**Comment by andrijeski** [comment]
Count me in as well! I was really happy when I found out about this project and saw that everything works just great :) 
Great work @jlai 💪

**Comment by mdouchin** [comment]
+1 with me !

**Comment by dot-mac** [comment]
Just checking in – I’m still using it

**Comment by dot-mac** [comment]
Still using

**Comment by dot-mac** [comment]
Still usingOn Sep 17, 2025, at 12:39 AM, mdouchin ***@***.***> wrote:﻿
+1 with me !

**Comment by dhdurgee** [comment]
Been using it ever since Google pulled their dashboard and this one became available.

**Comment by LawnPaul** [comment]
New user here!!!

**Reply by gramapp** [reply]
Welcome!

**Comment by jcarroll** [comment]
Found it today; recently started using my Alta HR again. Sad that Fitbit web went away. Glad to find this!

**Comment by dot-mac** [comment]
I am still using this, though not for its original intent. I use it to correct data in Google Health where possible, even if only by estimate. I appreciate the work on this project. Fitbit Charge 6.

**Comment by jtr13** [comment]
+1 Thank you!
